evap/grades/tests.py

```diff
@@ -22,19 +22,21 @@ def setUpTestData(cls):
         responsible = mommy.make(UserProfile, username="responsible", email="responsible@institution.example.com")
 
         cls.course = mommy.make(Course,
-            name_en="Test",
-            vote_start_date=datetime.date.today(),
-            participants=[
-                UserProfile.objects.get(username="student"),
-                UserProfile.objects.get(username="student2"),
-                UserProfile.objects.get(username="student3"),
-            ],
-            voters=[
-                UserProfile.objects.get(username="student"),
-                UserProfile.objects.get(username="student2"),
-            ]
-        )
-        contribution = Contribution(course=cls.course, contributor=responsible, responsible=True, can_edit=True, comment_visibility=Contribution.ALL_COMMENTS)
+                                name_en="Test",
+                                vote_start_date=datetime.datetime.now() - datetime.timedelta(10),
+                                vote_end_date=datetime.datetime.now() + datetime.timedelta(10),
+                                participants=[
+                                    UserProfile.objects.get(username="student"),
+                                    UserProfile.objects.get(username="student2"),
+                                    UserProfile.objects.get(username="student3"),
+                                ],
+                                voters=[
+                                    UserProfile.objects.get(username="student"),
+                                    UserProfile.objects.get(username="student2"),
```

Comment:
please put the students into variables and use those here.

...your fault if you're doing random formatting changes :)